core/rule-engine/rule-engine-components/src/main/resources/META-INF/services/org/apache/camel/component/email

```diff
@@ -0,0 +1 @@
+class=com.milesight.beaveriot.rule.components.email.EmailComponent
```

Comment:
Beans do not require the registration of the Camel Component

Reply:
Ok, I will delete it.